dev/MRTCore/packaging/Intellisense/Microsoft.Windows.ApplicationModel.Resources.Projection.xml

```diff
@@ -1,9 +1,241 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <doc>
   <assembly>
     <name>Microsoft.Windows.ApplicationModel.Resources.Projection</name>
   </assembly>
   <members>
-  </members>
-</doc>
+    <member name="T:Microsoft.Windows.ApplicationModel.Resources.KnownResourceQualifierName">
+      <summary>Contains names for common resource qualifiers such as language, system theme, and display scale.</summary>
+    </member>
+    <member name="P:Microsoft.Windows.ApplicationModel.Resources.KnownResourceQualifierName.Contrast">
+      <summary>Gets the name for a qualifier based on the system contrast settings.</summary>
+      <returns>The name for a qualifier based on the system contrast settings.</returns>
+    </member>
+    <member name="P:Microsoft.Windows.ApplicationModel.Resources.KnownResourceQualifierName.Custom">
+      <summary>Gets the name for a qualifier based on a custom value.</summary>
+      <returns>The name for a qualifier based a custom value.</returns>
+    </member>
+    <member name="P:Microsoft.Windows.ApplicationModel.Resources.KnownResourceQualifierName.DeviceFamily">
+      <summary>Gets the name for a qualifier based on the current device's family.</summary>
+      <returns>The name for a qualifier based on the current device's family.</returns>
+    </member>
+    <member name="P:Microsoft.Windows.ApplicationModel.Resources.KnownResourceQualifierName.HomeRegion">
+      <summary>Gets the name for a qualifier based on the current system home region.</summary>
```

Comment:
Re: "current *system* home region"
I wonder if this should say the "current *user's* home region" instead of system?

But I suspect that this is already coming from some pre-existing intellisense xml file (?).